### DISCUSSION TITLE
can not deploy by React project on github pages :: Error: .github#L1

### DISCUSSION THREAD

**Discussion by Ali503-7** [community/community#52485] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm trying to deploy my react project on GitHub-pages 

when I finish my project and create the .github/workflow/deploy.yml
and set the vite.config.js base: "/[repo name]/"

and then push the changes it gives me this error on action

 [Error: .github#L1](https://github.com/Ali503-7/My_Website/commit/0c91f6cb783995436fea3c4d2a2fe6e0155b7386#annotation_10408886368)
An unexpected error has occurred and we've been automatically notified. Errors are sometimes temporary, so please try again.

If the problem persists, please check whether the Actions service operates normally at https://githubstatus.com/. If not, please try again once the outage has been resolved.

Should you need to contact Support, please visit https://support.github.com/contact and include request ID: F1DE:979A:D550F1:DAB556:6434DC2E

how can I pass this problem

**Comment by airtower-luna** [comment]
The error message sounds like it's a GitHub problem, if it was a problem with your workflow you should see a message about the problem. So best try again to see if the problem went away, and if not [report the bug](https://support.github.com/contact/bug-report) to GitHub support (with the error ID as it says in the message), so they can take a look where us other users can't. :wink:

**Reply by Ali503-7** [reply]
Okay thx
